UVAtlasTool/Mesh.h

```diff
@@ -41,6 +41,9 @@ class Mesh
     // Methods
     void Clear();
 
+    void SetMTLFileName(const std::wstring& name) { mtlFileName = name; }
```

Comment:
For consistency with the rest of the interfaces, take ``const wchar_t*`` for the input parameters

Reply:
I don't follow. I'm not spotting other Setters to make this consistent with

Reply:
It's fine for this. I just usually prefer ``const wchar_t*`` for input parameters as it avoids forcing the caller to use ``std::wstring``. For this helper tool class, it doesn't really matter.